app/auth/user_management.py

```
@@ -0,0 +1,79 @@
+            "last_name": update_data.last_name,
+            "role": current_user.get("role", "user")
+        })
+        return {"message": "Profile updated successfully"}
+    except Exception as e:
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=str(e))
+
+
+@router.post("/bulk-action")
+async def bulk_user_action(
+    request: BulkUserRequest,
+    current_user: Dict[str, Any] = Depends(get_current_user)
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)